openfasoc/generators/ldo-gen/blocks/sky130hvl/pdn.tcl

```diff
@@ -13,12 +13,7 @@ add_global_connection -net {VSS} -pin_pattern {VGND} -ground
 add_global_connection -net {VSS} -pin_pattern {vgnd} -ground
 add_global_connection -net {VSS} -pin_pattern {VNB} -ground
 add_global_connection -net {VSS} -pin_pattern {vnb} -ground
```

Comment:
With the updated file. All 5 decaps are connected to VREF net.

Reply:
This would need to be updated based on our discussion today. @ayushman22git and @lulu9312 Please open a github issues like we discussed so we can track these.

Reply:
Yes.